service/web/adminserver.go

```diff
@@ -254,6 +254,15 @@ func (s adminServer) MigrateSubscriptions(ctx context.Context, subscriptions *me
 	return &emptypb.Empty{}, nil
 }
 
+func (s adminServer) SetupRetentionPolicy(ctx context.Context, _ *emptypb.Empty) (*metrov1.Topics, error) {
+	logger.Ctx(ctx).Infow("received request to set up retention policy")
```

Comment:
Is this log required, if yes we should add more details like topic

Reply:
@akashgoyal96 Request object doesn't have any topic parameter. We set up retention policy on all the dlq topics and return the updated topics in response